src/quoridor/service/gameservice.go

```diff
@@ -0,0 +1,48 @@
+package gameservice
```

Comment:
"gamemanager" for "gestionnaire de jeu" seems a better name IMO
We don't care that it's a service.

Reply:
I known that many people don't like "manager" name because it is "generic". But, in fact, it manage a game, no?